entrypoint.sh

```diff
@@ -25,3 +26,8 @@ fi
 unset JAVA_HOME
 
 sonar-scanner -Dsonar.projectBaseDir=${INPUT_PROJECTBASEDIR} ${INPUT_ARGS}
+
+_tmp_file=$(ls "${INPUT_PROJECTBASEDIR}/" | head -1)
+PERM=$(stat -c "%u:%g" "${INPUT_PROJECTBASEDIR}/$_tmp_file")
+
+chown -R $PERM "${INPUT_PROJECTBASEDIR}/"
```

Comment:
What are you trying to target here? `.scannerwork`? Because:

- `ls` will not show hidden dotfiles by default (unless you use `-a`)
- If you are trying to target `.scannerwork`, you are making the assumption that it's the only dotfile in the input dir (because of `head -1`), but that's not always the case. For instance, we have `.github/` in this repository.

Reply:
i am using the first file in the projects base dir to extract the owner:group from it. it does not need to be a hidden file as all permissions that are in the directory at that time are coming from a GHA run. 
after that i am setting the ownership to this extracted uid/gid for all files in the projects base dir, including the .scannerwork. 

this is the result:
![image](https://user-images.githubusercontent.com/64127335/156132929-bae49560-c05b-4467-ae1c-09afc64bcc6b.png)

Reply:
Ah, misread your code 🤦 . My bad!